util/encode_source.pl

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env perl
 
 use warnings;
+use File::Basename;
 
 # Incoming arguments
 my $sourceFile	= $ARGV[0];
 my $destFile	= $ARGV[1];
-my $varName		= $ARGV[2];
+my $varName	= basename($ARGV[2]);
```

Comment:
I'll add something to save you doing it @livecodefraser ... I've already merged the PR ;-)